cmd/agent/subcommands/streamlogs/command.go

```diff
@@ -33,18 +37,30 @@ type cliParams struct {
 	filters diagnostic.Filters
 }
 
+type StreamLogParams struct {
```

Comment:
Any reason not to re-use `cliParams`?

Reply:
i put it as a different struct since i'm not sure beside output and duration, if we would add more arguments in the future